package.json

```diff
@@ -34,100 +34,105 @@
-    "@tsconfig/ember": "^2.0.0",
-    "@types/ember": "^4.0.3",
-    "@types/ember__application": "^4.0.5",
-    "@types/ember__array": "^4.0.3",
-    "@types/ember__component": "^4.0.13",
-    "@types/ember__controller": "^4.0.4",
-    "@types/ember__debug": "^4.0.3",
-    "@types/ember__destroyable": "^4.0.1",
-    "@types/ember__engine": "^4.0.4",
-    "@types/ember__error": "^4.0.2",
-    "@types/ember__object": "^4.0.5",
-    "@types/ember__polyfills": "^4.0.1",
-    "@types/ember__routing": "^4.0.12",
-    "@types/ember__runloop": "^4.0.2",
-    "@types/ember__service": "^4.0.2",
-    "@types/ember__string": "^3.0.10",
-    "@types/ember__template": "^4.0.1",
-    "@types/ember__test": "^4.0.1",
-    "@types/ember__test-helpers": "^2.9.1",
-    "@types/ember__utils": "^4.0.2",
-    "@types/ember-data": "^4.4.11",
-    "@types/ember-data__adapter": "^4.0.2",
-    "@types/ember-data__model": "^4.0.1",
-    "@types/ember-data__serializer": "^4.0.2",
-    "@types/ember-data__store": "^4.0.3",
-    "@types/ember-qunit": "^6.1.1",
-    "@types/ember-resolver": "^9.0.0",
-    "@types/node": "^18.14.5",
-    "@types/qunit": "^2.19.4",
-    "@types/rsvp": "^4.0.4",
-    "@typescript-eslint/eslint-plugin": "^5.61.0",
-    "@typescript-eslint/parser": "^5.61.0",
+    "@release-it-plugins/lerna-changelog": "^7.0.0",
+    "@tsconfig/ember": "^3.0.8",
+    "@types/ember": "^4.0.11",
+    "@types/ember__application": "^4.0.11",
+    "@types/ember__array": "^4.0.10",
+    "@types/ember__component": "^4.0.22",
+    "@types/ember__controller": "^4.0.12",
+    "@types/ember__debug": "^4.0.8",
+    "@types/ember__destroyable": "^4.0.5",
+    "@types/ember__engine": "^4.0.11",
+    "@types/ember__error": "^4.0.6",
+    "@types/ember__modifier": "^4.0.9",
+    "@types/ember__object": "^4.0.12",
+    "@types/ember__polyfills": "^4.0.6",
+    "@types/ember__routing": "^4.0.22",
+    "@types/ember__runloop": "^4.0.10",
+    "@types/ember__service": "^4.0.9",
+    "@types/ember__string": "^3.16.3",
+    "@types/ember__template": "^4.0.7",
+    "@types/ember__test": "^4.0.6",
+    "@types/ember__test-helpers": "^2.9.3",
+    "@types/ember__utils": "^4.0.7",
+    "@types/ember-data": "^4.4.16",
+    "@types/ember-data__adapter": "^4.0.6",
+    "@types/ember-data__model": "^4.0.5",
+    "@types/ember-data__serializer": "^4.0.6",
+    "@types/ember-data__store": "^4.0.7",
+    "@types/ember-qunit": "^6.1.3",
+    "@types/ember-resolver": "^9.0.2",
+    "@types/node": "^22.10.6",
+    "@types/qunit": "^2.19.12",
+    "@types/rsvp": "^4.0.9",
+    "@typescript-eslint/eslint-plugin": "^8.20.0",
+    "@typescript-eslint/parser": "^8.20.0",
     "broccoli-asset-rev": "^3.0.0",
-    "concurrently": "^8.0.1",
-    "date-fns": "^2.29.3",
-    "ember-auto-import": "^2.6.3",
-    "ember-cli": "~4.12.1",
-    "ember-cli-app-version": "^6.0.0",
+    "concurrently": "^8.2.2",
+    "date-fns": "^4.1.0",
+    "ember-auto-import": "^2.10.0",
+    "ember-cli": "~6.1.0",
```

Comment:
Strange, did you bump this to 6.1 yourself? Ideally this matches the ember-source version (unless there is a good reason not to).